extensions/mssql/src/contracts.ts

```diff
@@ -1024,3 +1024,21 @@ export namespace GetSqlMigrationAssessmentItemsRequest {
 }
 
 // ------------------------------- <Sql Migration> -----------------------------
+
+// ------------------------------- < Table Designer > ------------------------------------
+
+export interface ProcessTableDesignerEditRequestParams {
+	tableInfo: azdata.designers.TableInfo,
+	edit: azdata.designers.DesignerEdit,
```

Comment:
name of this object is a little unclear.  Maybe something like `tableChangeInfo` or something more descriptive.